src/Form/ActionForm.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Drupal\entity_reference_actions\Form;
```

Comment:
The class should have a different name since it is not a form, maybe _EntityActions_ _EntitiyOperations_ or something along the line.
Also namespace and directory should not include _Form_, since again its not a form.

Reply:
Opened #6